cogboard-app/src/main/kotlin/com/cognifide/cogboard/storage/StorageInitializer.kt

```diff
@@ -0,0 +1,27 @@
+package com.cognifide.cogboard.storage
+
+import com.cognifide.cogboard.widget.type.ExampleWidget
+import io.vertx.core.logging.LoggerFactory
+import java.io.File
+
+class StorageInitializer {
+
+    companion object {
+        private val LOGGER = LoggerFactory.getLogger(ExampleWidget::class.java)
+
+        fun createIfDoesNotExist(configPath: String) {
+            val fileNotExists = !File(configPath).exists()
```

Comment:
we don't need this `val` move it directly to `if ()`

Reply:
moved